src/world/CMakeLists.txt

```diff
@@ -149,11 +194,15 @@ if (WIN32 AND NOT USE_PCH)
 endif ()
 
 if (APPLE)
-    target_link_libraries(${PROJECT_NAME} c++)
+    LIST(APPEND WORLD_LINK_EXTRA_LIBS c++)
 elseif (CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD")
-    target_link_libraries(${PROJECT_NAME} c++experimental)
+    LIST(APPEND WORLD_LINK_EXTRA_LIBS c++experimental)
```

Comment:
Is Fedora/FreeBSD/kFreeBSD still relevant "c++experimental"? I just don't know..

Reply:
actually i'm unsure, i think this can be removed but i don;t have freebsd installed by this moment, I cannot review. For Fedora - its not used. By default I just reused existing code (well, optimized, but it does same as previous code)